lib/data_update_scripts/20220906190412_coerce_unicorn_reactions_to_likes.rb

```diff
@@ -0,0 +1,29 @@
+module DataUpdateScripts
+  class CoerceUnicornReactionsToLikes
+    def run
+      # Find all unicorn reactions where there exists a like reaction on the same reactable
+      # by the same user, and delete them (These would cause errors when we try to flip all
+      # the unicorn reactions to a like)
+      ActiveRecord::Base.connection.execute(
+        <<-SQL,
+          DELETE FROM reactions
```

Comment:
Everything I've been able to find indicates this will be a row level lock in the database, instead of a table level lock. We should be deleting about 700k out of 9 million reaction records here